projects/plugins/jetpack/_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php

```diff
@@ -164,7 +173,13 @@ public function can_request() {
 	}
 }
 
+/**
+ * Interact with multiple modules at once (list or activate).
+ *
+ * // phpcs:disable Generic.Files.OneObjectStructurePerFile.MultipleFound
```

Comment:
Would it be worth splitting these into separate files, properly named, with this file just having `require_once`s of them all as a shortcut and/or for back-compat?

Reply:
Yeah, I think it'd be worth doing. I can tackle that in a follow-up PR.